code/game/objects/items/weapons/tools.dm

```diff
@@ -435,26 +442,37 @@
 	toggle(user)
 
 //Returns the amount of fuel in the welder
-/obj/item/tool/weldingtool/proc/get_fuel()
-	return reagents.get_reagent_amount(FUEL)
+/obj/item/tool/weldingtool/proc/get_fuel(var/type = 0)
+	if(type == FUEL)
+		return reagents.get_reagent_amount(FUEL)
+	else if(type == PLASMA)
+		return reagents.get_reagent_amount(PLASMA)
+	else
+		return reagents.get_reagent_amount(FUEL) + reagents.get_reagent_amount(PLASMA)
```

Comment:
```suggestion
    return reagents.get_reagent_amount(FUEL) + reagents.get_reagent_amount(PLASMA)
```
keep it simple

Reply:
this suggested change won't work as it makes the argument unused